rfc/scope_manager.md

```diff
@@ -1,6 +1,6 @@
 # Scope Manager
 
-**Current State:** Draft  
+**Current State:** Test
```

Comment:
Hello @carlosalberto,
I think you should not remove two ending spaces here.
They are [required to split lines](https://daringfireball.net/projects/markdown/syntax#p).
> When you do want to insert a \<br /\> break tag using Markdown, you end a line with two or more spaces, then type return.

[Currently](https://github.com/opentracing/specification/blob/434e6764129e0eb02d27fd096191ca52cc8bb2c9/rfc/scope_manager.md), you have `Current State` and `Author` on the same line.

Reply:
Wouldn't a blank line between them do the same? Linters do not like trailing spaces (they even show up as red in git diff).

Alternatively, in this case it's appropriate to us bullet points for the metadata.

Reply:
> Wouldn't a blank line between them do the same?

No, it wouldn't. The result won't be the same.

Reply:
> Linters do not like trailing spaces

I know, but it's an issue of those linters and `git`.
It's not a `Markdown` issue.